src/ResizeSensor.js

```diff
@@ -83,9 +83,13 @@
      * @constructor
      */
     var ResizeSensor = function(element, callback) {
-       
+
+        if (element === undefined || element === null) {
+            return;
```

Comment:
The question is should we not rather throw an error when this is the case? I mean, the class requires an element, and when not given its clearly an error. Can you adjust that tot `throw new Error('No element given')`.

Reply:
I don't think throw is a good idea here
It basically kills the program instead of having just no resize sensor, since you cant expect that in real live code there is a catch after the resizesensor initialisation
Wouldn't console.error be enough?

Reply:
Its an invalid usage of the class, which should always lead the an Error being thrown, that's why a error system exists. :)

Reply:
Yeah but you cannot expect nice error handling everywhere.
I would always prefer the resize sensor not to work instead of killing the whole program just because i forgot to insert a catch after every resizesensor instance.
What speaks against a console error?
It is visible and doesn't require specific error handling.